k8s/upgrade/src/preflight_validations.rs

```diff
@@ -185,7 +199,7 @@ pub async fn upgrade_path_validation(namespace: &str) -> error::Result<()> {
 
     if unsupported_versions.contains(&source) {
         let mut invalid_source_list: String = String::new();
```

Comment:
nit: I think Default::default() makes it a bit more obvious instead of String::new().